lib/fluent/plugin/out_suppress.rb

```diff
@@ -1,45 +1,31 @@
-# -*- coding: utf-8 -*-
-module Fluent
-  class SuppressOutput < Output
+require "fluent/plugin/output"
+
+module Fluent::Plugin
+  class SuppressOutput < Fluent::Plugin::Output
     include Fluent::HandleTagNameMixin
```

Comment:
Sorry for delay.
No we cannot use inject plugin helper instead of this mixin.
`Fluent::HandleTagNameMixin` does not inject any fields into a record.
It modifies only tag.

See also http://docs.fluentd.org/v0.14/articles/plugin-update-from-v12